replibyte/src/config.rs

```diff
@@ -210,7 +233,13 @@ impl TransformerTypeConfig {
                 table_name,
                 column_name,
             )),
-            TransformerTypeConfig::Redacted => Box::new(RedactedTransformer::new(
+            TransformerTypeConfig::Redacted(options) => Box::new(RedactedTransformer::new(
+                database_name,
+                table_name,
+                column_name,
+                options.clone(),
```

Comment:
@fabriceclementz I think it's good to clone here. At least it's easy to maintain. We'll see if it's a problem in the long run but I think it should be ok.

Reply:
👌